### PR TITLE
Feat/aias5

### PR DESCRIPTION
**Pull request by sylvaingaudan**
Upgrade to new AIAS 5
- enrich
- zarr
- titiler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,18 @@ To setup an ARLAS EO catalog:
...
 
+## Data from GEODES
+
+You can register data from GEODES:
+
+```shell
+docker run --rm --network arlas-net gisaia/stac-geodes:latest add https://geodes-portal.cnes.fr/api/stac/items http://airs-server:8000/airs geodes S2L1C --start-date "2023-04-05T08:58:40.737+00:00" --max 1000
+```
+
+This will register in the `airs_geodes` index the first 1000 `S2L1C` data that acquired after 2023-04-05T08:58. Then, you can create the catalog:
```
Comment (QuCMGisaia) [inline README.md]: data that were

File: conf/aias.env [hunk 2]
```diff
@@ -1,7 +1,7 @@
 AIRS_CATALOG=catalog
 AIRS_COLLECTION=main
 AIRS_INDEX_ENDPOINT_URL=http://elasticsearch:9200
-AIRS_LOGGER_LEVEL=INFO
+AIRS_LOGGER_LEVEL=DEBUG
```
Comment (QuCMGisaia) [inline conf/aias.env]: Is that wanted?

File: conf/aias.env [hunk 3]
```diff
@@ -59,7 +65,7 @@ APROC_INDEX_ENDPOINT_URL=http://elasticsearch:9200
 APROC_INDEX_NAME=aproc_downloads
 APROC_RESOURCE_ID_HASH_STARTS_AT=1
 ARLAS_URL_SEARCH="http://arlas-server:9999/arlas/explore/{collection}/_search?f=id:eq:{item}"
-
+AGATE_LOGGER_LEVEL=DEBUG
```
Comment (QuCMGisaia) [inline conf/aias.env]: Is that wanted?

**Review by QuCMGisaia — APPROVED** [review]

File: docker_compose_services_simple.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ List of volumes:
 ### Service arlas-server
 Description: ARLAS Server is the geo-analytic engine of the ARLAS Exploration Stack
 
-Image: `ARLAS_SERVER_VERSION` with `gisaia/arlas-server:26.0.1` in `conf/versions.env`
+Image: `ARLAS_SERVER_VERSION` with `gisaia/arlas-server:26.0.2` in `conf/versions.env`
```
Comment (MohamedHamouGisaia) [inline docker_compose_services_simple.md:34]: 26.0.3

**Pull request merged**